### DISCUSSION TITLE
Maven: Allow for different name of POM file

### DISCUSSION THREAD

**Discussion by fibsifan** [aquasecurity/trivy#6209] [Ideas]
### Description

Hello,

i'm working on a project where the name of the pom-file is not pom.xml. Maven allows to provide the name of this file with the -f option.

I tried Analyzing a Source-Code-Repo on File with trivy but it didn't recognized the alternativly named pom-files.

Could this be added as a command line feature?

### Target

Filesystem

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
`--file-patterns pom:my-pom.xml` should work. Please let me know if it doesn't work.
https://aquasecurity.github.io/trivy/v0.49/docs/configuration/skipping/#file-patterns

**Comment by fibsifan** [comment]
That works perfectly. Sorry I overlooked that in the Documentation
